ci-operator/step-registry/single-node/conf/aws/graceful-shutdown/single-node-conf-aws-graceful-shutdown-commands.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SINGLE_NODE_AWS_INSTANCE_TYPE="m5d.2xlarge"
```

Comment:
@omertuc is it the correct instance size?

Reply:
We took the instance size from the original: https://github.com/openshift/release/blob/5fe1328b1bd7593ed71f942eacbda18ea95ced86/ci-operator/step-registry/single-node/conf/aws/single-node-conf-aws-commands.sh#L7